src/box/vinyl.c

```diff
@@ -1038,6 +1038,28 @@ vinyl_space_invalidate(struct space *space)
 	 */
 	bool unused;
 	vy_tx_manager_abort_writers_for_ddl(env->xm, space, &unused);
+
```

Comment:
By the way, `vy_tx_manager_abort_writers_for_ddl` seems to be broken since it retrieves primary key from the space that is being invalidated, but the index can be moved to the new space and the function will use a dummy one.

Reply:
Good catch! I decided to rewrite `vy_tx_manager_abort_writers_for_ddl` to use `txn->stmts` instead. This should fix the issue. See #10916.